README.md

```diff
@@ -130,43 +130,48 @@ With your own parameters:
 
 <h2 align="center">Main Features</h2>
 
-**Splitting video by scenes for parallel encoding** because AV1 encoders are currently not very good at multithreading and encoding is limited to a very limited number of threads.
+Av1an allows for **splitting input video by scenes for parallel encoding** to improve encoding performance because most AV1 encoders are currently not very good at multithreading and encoding is limited to a very limited number of threads.
```

Comment:
```suggestion
Av1an allows for **splitting input video by scenes for parallel encoding** to improve encoding performance, because most AV1 encoders are currently not very good at multithreading and encoding is limited to a very limited number of threads.
```